src/domain/event/eventForm/EventForm.tsx

```diff
@@ -170,4 +184,96 @@ const EventForm = () => {
   );
 };
 
+const ImageSelectedForm: React.FC<{
+  image: string;
+  setFieldValue: (
+    field: string,
+    value: any,
+    shouldValidate?: boolean | undefined
+  ) => void;
+}> = ({ setFieldValue, image }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <ImagePreview image={image} />
+      <div className={styles.imageSelectRow}>
+        <div className={styles.imageSelect}>
+          <DeleteImageButton
+            onClick={() => {
+              ['image', 'photographer', 'imageAltText'].forEach((field) => {
+                setFieldValue(field, '');
+              });
+            }}
+          />
+        </div>
+        <div>
+          <FormGroup>
+            <Field
+              labelText={t('eventForm.basicInfo.photographer')}
+              name="photographer"
+              component={TextInputField}
+            />
+          </FormGroup>
+          <FormGroup>
+            <Field
+              labelText={t('eventForm.basicInfo.labelImageAltText')}
+              name="imageAltText"
+              helperText={t('eventForm.basicInfo.imageAltTextHelp')}
+              component={TextInputField}
+            />
+          </FormGroup>
+        </div>
+      </div>
+    </>
+  );
+};
+
+const SelectImageForm: React.FC = () => {
+  const { t } = useTranslation();
```

Comment:
Could be moved to own file. SelectImageForm is little bit misleading name because this component is not form